neptune/new/handler.py

```diff
@@ -144,5 +258,166 @@ def pop(self, path: str, wait: bool = False) -> None:
+                Defaults to `False`.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
+        """
+        raise NeptuneException('Should be never called.')
+
+    def fetch(self):
+        """Fetches field value from Neptune servers.
+
+        Available for following field types (`Field types docs page`_):
+            * `Integer`
+            * `Float`
+            * `Boolean`
+            * `String`
+            * `DateTime`
+            * `StringSet`
+
+        Returns:
+            Value stored in the field in form of `set` or sigle value.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
+        """
+        raise NeptuneException('Should be never called.')
+
+    def fetch_last(self):
+        """Fetches last value stored in the series from Neptune servers.
+
+        Available for following field types (`Field types docs page`_):
+            * `FloatSeries`
+            * `StringSeries`
+
+        Returns:
+            Fetches last value stored in the series from Neptune servers.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
+        """
+        raise NeptuneException('Should be never called.')
+
+    def fetch_values(self, include_timestamp: Optional[bool] = True):
+        """Fetches all values stored in the series from Neptune servers.
+
+        Available for following field types (`Field types docs page`_):
+            * `FloatSeries`
+            * `StringSeries`
+
+        Args:
+            include_timestamp (bool, optional): Whether the fetched data should include the timestamp field.
+                Defaults to `True`.
+
+        Returns:
+            ``Pandas.DataFrame``: containing all the values and their indexes stored in the series field.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
+        """
+        raise NeptuneException('Should be never called.')
+
+    def delete_files(self, paths: Union[str, Iterable[str]], wait: bool = False) -> None:
+        """Delete the file or files specified by paths from the `FileSet` stored on the Neptune servers.
+
+        Available for following field types (`Field types docs page`_):
+            * `FileSet`
+
+        Args:
+            paths (str or collection of str): `Path` or paths to files or folders to be deleted.
+                Note that these are paths relative to the FileSet itself e.g. if the `FileSet` contains
+                file `example.txt`, `varia/notes.txt`, `varia/data.csv` to delete whole subfolder you would pass
+                varia as the argument.
+            wait (bool, optional): If `True` the client will wait to send all tracked metadata to the server.
+                This makes the call synchronous.
+                Defaults to `None`.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
```

Comment:
Let's link explicitly to https://docs.neptune.ai/api-reference/field-types#delete_files